bezier_library/src/bezier_library.cpp

```diff
@@ -81,6 +82,23 @@ bool Bezier::getTranslationMode()
   return use_translation_mode_;
 }
 
+void Bezier::SetSurfacingOn(){
+  ROS_INFO_STREAM("WARM : Running in surfacing mode");
+  surfacing_ = true;
+}
+
+void Bezier::SetSurfacingOff(){
+  surfacing_ = false;
+}
+
+void Bezier::SetSurfacing(const bool surfacing_mode){
+  surfacing_ = surfacing_mode;
+}
+
+bool Bezier::GetSurfacing() const{
+  return surfacing_;
+}
```

Comment:
Please use the same coding style as the other members (spaces, parenthesis, line returns).
We (approximately...) use the [ROS coding style](http://wiki.ros.org/CppStyleGuide).